TLSharp.Core/Network/TcpTransport.cs

```diff
@@ -1,7 +1,8 @@
 ﻿using System;
-using System.Linq;
+using System.Diagnostics;
 using System.Net;
 using System.Net.Sockets;
+using System.Reflection;
```

Comment:
The sample of code does not work for me :-/
I do not know what to do with it...